[2023-01-08 11:32:04,160] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_scoring_training_pipeline.training_model manual__2023-01-08T11:31:51.070409+00:00 [queued]>
[2023-01-08 11:32:04,165] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_scoring_training_pipeline.training_model manual__2023-01-08T11:31:51.070409+00:00 [queued]>
[2023-01-08 11:32:04,165] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-08 11:32:04,165] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-01-08 11:32:04,165] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-08 11:32:04,176] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): training_model> on 2023-01-08 11:31:51.070409+00:00
[2023-01-08 11:32:04,184] {standard_task_runner.py:52} INFO - Started process 23343 to run task
[2023-01-08 11:32:04,190] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_scoring_training_pipeline', 'training_model', 'manual__2023-01-08T11:31:51.070409+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/Lead_scoring_training_pipeline/.ipynb_checkpoints/lead_scoring_training_pipeline-checkpoint.py', '--cfg-path', '/tmp/tmps87sd1qo', '--error-file', '/tmp/tmpw_d84a2_']
[2023-01-08 11:32:04,190] {standard_task_runner.py:80} INFO - Job 124: Subtask training_model
[2023-01-08 11:32:04,237] {task_command.py:371} INFO - Running <TaskInstance: Lead_scoring_training_pipeline.training_model manual__2023-01-08T11:31:51.070409+00:00 [running]> on host 1f037b111ba5
[2023-01-08 11:32:04,291] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Lead_scoring_training_pipeline
AIRFLOW_CTX_TASK_ID=training_model
AIRFLOW_CTX_EXECUTION_DATE=2023-01-08T11:31:51.070409+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-08T11:31:51.070409+00:00
[2023-01-08 11:32:08,973] {logging_mixin.py:115} WARNING - /opt/conda/lib/python3.8/site-packages/sklearn/utils/validation.py:72 DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
[2023-01-08 11:32:13,402] {logging_mixin.py:115} WARNING - Registered model 'LightGBM' already exists. Creating a new version of this model...
[2023-01-08 11:32:13,437] {logging_mixin.py:115} WARNING - 2023/01/08 11:32:13 INFO mlflow.tracking._model_registry.client: Waiting up to 300 seconds for model version to finish creation.                     Model name: LightGBM, version 2
[2023-01-08 11:32:13,437] {logging_mixin.py:115} WARNING - Created version '2' of model 'LightGBM'.
[2023-01-08 11:32:13,649] {python.py:173} INFO - Done. Returned value was: None
[2023-01-08 11:32:13,657] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=Lead_scoring_training_pipeline, task_id=training_model, execution_date=20230108T113151, start_date=20230108T113204, end_date=20230108T113213
[2023-01-08 11:32:13,695] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-08 11:32:13,712] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
